batch_requests/views.py

```diff
@@ -77,7 +77,10 @@ def timeout_result_handler(future, timeout=None):
 def handle_sub_response_body(response):
     body = getattr(response, "rendered_content", None) or response.content
     if body and _settings.DESERIALIZE_RESPONSES:
-        return json.loads(body)
+        try:
+            return json.loads(body)
+        except ValueError:
+            pass  # fall through and just return body
```

Comment:
This bit is, essentially, the point of the ticket. We were failing because we were trying to deserialize **everything** and not catching any exceptions.

Now, we'll continue to try to deserialize everything, but fail to just using the body if it doesn't deserialize cleanly, e.g. is not valid JSON.